cmd/libs/go2idl/deepcopy-gen/main.go

```diff
@@ -34,6 +34,19 @@ func main() {
 	// Override defaults. These are Kubernetes specific input locations.
 	arguments.InputDirs = []string{
 		"k8s.io/kubernetes/pkg/api",
+		"k8s.io/kubernetes/pkg/api/v1",
+		"k8s.io/kubernetes/pkg/apis/authorization",
+		"k8s.io/kubernetes/pkg/apis/authorization/v1beta1",
+		"k8s.io/kubernetes/pkg/apis/autoscaling",
+		"k8s.io/kubernetes/pkg/apis/autoscaling/v1",
+		"k8s.io/kubernetes/pkg/apis/batch",
+		"k8s.io/kubernetes/pkg/apis/batch/v1",
+		"k8s.io/kubernetes/pkg/apis/componentconfig",
+		"k8s.io/kubernetes/pkg/apis/componentconfig/v1alpha1",
+		"k8s.io/kubernetes/pkg/apis/extensions",
+		"k8s.io/kubernetes/pkg/apis/extensions/v1beta1",
+		"k8s.io/kubernetes/pkg/apis/metrics",
+		"k8s.io/kubernetes/pkg/apis/metrics/v1alpha1",
```

Comment:
Yeah - I was also planing to do something like that. So much things to do, so little time :)